spec/rubocop/cop/style/inverse_methods_spec.rb

```diff
@@ -50,6 +50,20 @@
     expect_no_offenses('!!foo.reject { |e| !e }')
   end
 
+  it 'allows an inverse method in a block with next' do
+    expect_no_offenses(<<-RUBY.strip_indent)
```

Comment:
Could you rebase using latest master branch and change it to the following code?

```diff
-expect_no_offenses(<<-RUBY.strip_indent)
+expect_no_offenses(<<~RUBY)
```

Reply:
done!